mscore/resourceManager.cpp

```diff
@@ -499,20 +500,52 @@ static std::vector<std::pair<QString, QString>> getAttachments(const QByteArray&
             while (!xml.atEnd() && xml.name() == "tr") {
                   for (int i = 0; i < 3; i++) // enter <td>, <span>, <a>
                         xml.readNextStartElement();
-                  file_urls.push_back(make_pair(xml.readElementText(), xml.attributes().value("href").toString()));
+                  PluginPackageLink link = { ATTACHMENT,xml.attributes().value("href").toString() };
+                  //link.hint = xml.readElementText();
+                  link.score = CompatEstimate(xml.readElementText());
+                  file_urls.push_back(link);
                   for (int i = 0; i < 3; i++) // exit <a>, <span>, <td>
                         xml.skipCurrentElement();
                   xml.readNextStartElement();
                   }
             }
       else {
             // search for all links from musescore.org
-            QRegularExpression musescore_link_patt("https://musescore.org/sites/musescore.org/files/\\d+\\-\\d+/([\\w\\-]+)");
+            QRegularExpression musescore_link_patt("<a href=\"(https://musescore.org/sites/musescore.org/files/(\\d+\\-\\d+/)?[\\w\\-\\.]+)\".*?>(.*?)</a>");
             auto it = musescore_link_patt.globalMatch(html_raw);
             while (it.hasNext()) {
                   QRegularExpressionMatch match = it.next();
-                  file_urls.push_back(make_pair(match.captured(1), match.captured(0)));
+                  QString hint_text = match.captured(2);
+                  QString hint_debug = match.captured(0);
+                  int start_idx = match.capturedStart(0);
+                  int end_idx = match.capturedEnd();
+                  int newline_start = start_idx, newline_end = end_idx;
+                  while (newline_start>0 && html_raw[newline_start-1] != '\n')
+                        newline_start--;
+                  while (newline_end < html_raw.length() && html_raw[newline_end] != '\n')
+                        newline_end++;
+                  QString curr_line = html_raw.mid(newline_start, newline_end - newline_start + 1);
+                  int score = CompatEstimate(curr_line);
+                  PluginPackageLink link = { ATTACHMENT,match.captured(1),newline_start,curr_line,score };
```

Comment:
Maybe `PluginPackageLink link = { {ATTACHMENT}, {match.captured(1)}, [newline_start}, {curr_line}, {score} };` works better?

Reply:
Or simply drop the `=`, `PluginPackageLink link{ATTACHMENT, match.captured(1), newline_start, curr_line, score};`
Or `PluginPackageLink link{ {ATTACHMENT, match.captured(1), newline_start, curr_line, score}; };`